ansible/group_vars/all/main.yml

```diff
@@ -0,0 +1,7 @@
+---
+# vars file for create-ec2-instances
+region_name: 'us-east-1'
+key_name: 'piemdbTesting'
+ami_id: 'ami-0323c3dd2da7fb37d'
```

Comment:
Hard coding `ami_id` doesn't seem to be a long term solution, but should be fine for now unless there's a way to get it automatically